tchannel-core/src/main/java/com/uber/tchannel/api/TChannel.java

```diff
@@ -162,6 +168,8 @@ public void shutdown() throws InterruptedException {
 
         private final String service;
         private final ChannelManager channelManager = new ChannelManager();
+        private ExecutorService executorService = new ForkJoinPool();
+        private int maxQueuedRequests = Runtime.getRuntime().availableProcessors() * 5;
```

Comment:
I can add it as a constant, this is not a magic number per se it more like sane defaults